tests/compiler/std/string.ts

```diff
@@ -13,6 +13,11 @@ assert("\xDFab" == "ßab");
 
 assert(str.length == 16);
 assert(str.charCodeAt(0) == 0x68);
+assert(str.at(15) == str.charAt(15));
+assert(str.at(-1) == str.charAt(str.length - 1));
+assert(str.at(-str.length) == "h");
+assert(str.at(str.length) == "");
```

Comment:
Wouldn't this one produce `undefined` in JS, and should throw in AS?

Reply:
Yes, in JS it produce [`undefined`](https://tc39.es/proposal-relative-indexing-method/#sec-string.prototype.at). So you suggest throw an error?

Reply:
I believe these are different. `.charAt` returns an empty string in JS for example, but `operator[]` would return `undefined`. So I guess both `operator[]` and `.at` should throw, but `.charAt` remain as-is.

Reply:
I see. But this will minor breaking changes